File: class/Person.cpp
// Person.cpp
#include "Person.h"
Person::Person(string name,
               char gender,
               int age)
   : name(name), gender(gender), age(age) {}
string Person::getName() const
  { return name; }
char Person::getGender() const
  { return gender; }
int Person::getAge() const
  { return age; }
void Person::setName(string name)
  { this->name = name; }
void Person::setGender(char gender)
  { this->gender = gender; }
void Person::setAge(int age)
  { this->age = age; }
void Person::print() const {
  cout << "Name: " << name << endl;
  cout << "Gender: " << gender << endl;
  cout << "Age: " << age << endl;
}

File: class/Person.java
// Person.java
public class Person {
  private String name;
  private char gender;
  private int age;
  public Person(String name, char gender, int age) {
    this.name = name;
    this.gender = gender;
    this.age = age;
  }
  public String getName() { return name; }
  public char getGender() { return gender; }
  public int getAge() { return age; }
  public void setName(String n) { name = n; }
  public void setGender(char g) { gender = g; }
  public void setAge(int a) { age = a; }
  public void print() {
    System.out.println("Name: " + name + "\nGender: " + gender + "\nAge: " + age);
  }
} // No semi-colon here